components/Instructions/Instructions.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+
+export interface InstructionsProps {
+    number: number;
+    brief: string;
+    example: string;
+    inputOutput: InputOutputProps;
+    jsTips: string;
+}
+
+export interface InputOutputProps {
+    executionTimeLimit: string;
+    inputType: string;
+    inputDescription: string;
+    input: string | number | string[] | number[];
+    outputType: string;
+    outputDescription: string;
+    expectedOutput: string | number | string[] | number[];
+}
+
+function buildInstructionsProps(testX: InstructionsProps) {
+    return testX;
+}
+
+export function GetInstructionsProps(number: number, brief: string, example: string, inputOutput: InputOutputProps, jsTips: string): InstructionsProps {
+    let InstructionsX: InstructionsProps;
+    buildInstructionsProps(InstructionsX = {
+        number: number,
+        brief: brief,
+        example: example,
+        inputOutput: inputOutput,
+        jsTips: jsTips
+    });
+    return InstructionsX;
+}
+
+function buildInputOutputProps(inputOutputX: InputOutputProps) {
+    return inputOutputX;
+}
+
+export function GetInputOutputProps(executionTimeLimit: string, inputType: string, input: string | number | string[] | number[], inputDescription: string, outputType: string, outputDescription: string, expectedOutput: string | number | string[] | number[]): InputOutputProps {
+    let InputOutputX: InputOutputProps;
+    buildInputOutputProps(InputOutputX = {
+        executionTimeLimit: executionTimeLimit,
+        inputType: inputType,
+        inputDescription: inputDescription,
+        input: input,
+        outputType: outputType,
+        outputDescription: outputDescription,
+        expectedOutput: expectedOutput
+    });
+    return InputOutputX;
+}
+
+type InstructionsBlockProps = InstructionsProps;
+const InstructionsBlock = ({inputOutput, brief, jsTips, example, number}: InstructionsBlockProps): JSX.Element => (
+    <div>
+        <p>{brief}</p>
+        <br/>
+        <h2>Example</h2>
+        <p>{example}</p>
+        <br/>
+        <h2>Input/Output</h2>
+        <ul>
+            <li><b>[execution time limit] {inputOutput.executionTimeLimit} (js)</b></li>
+            <br/>
+            <li><b>[input] {inputOutput.inputType}</b><br/>{inputOutput.inputDescription}</li>
+            <br/>
+            <li><b>[output] {inputOutput.outputType}</b><br/>{inputOutput.outputDescription}</li>
+        </ul>
+        <br/>
+        <h2>[JavaScript (ES6)] Syntax Tips</h2>
+        <p>{jsTips}</p>
```

Comment:
are these part of the designs? Do we need them?

Reply:
As above, they were included in the test Sasha sent over so thought I should include them

Reply:
They are in the existing copy... but not in the designs for what we said we'd build.
I'd be tempted not to try adding in stuff that we didn't promise.

If we do put them in, then we should style them properly with syntax highlighting etc.